en/frames_multicopter/omnicopter.md

```diff
@@ -0,0 +1,107 @@
+# Omnicopter
+
+An Omnicopter is a multicopter that can provide thrust in all directions (6 degrees of freedom).
+This allows it to move in any direction without having to tilt, and it can hover at an arbitrary tilt angle.
+All of this is achieved by arranging the motor positions and thrust axis in specific ways:
+
+<img src="../../assets/airframes/multicopter/omnicopter/frame.jpg" style="display: block; margin: 0 auto"/>
+
+This build follows the original design from [Brescianini, Dario, and Raffaello D'Andrea](https://www.youtube.com/watch?v=sIi80LMLJSY).
+
+
+## Simulation
+There is an omnicopter gazebo simulation target:
+```sh
+make px4_sitl gazebo_omnicopter
+```
+![Gazebo Sim](../../assets/airframes/multicopter/omnicopter/gazebo.png)
+
+## Bill of materials
+The components needed for this build are:
+- Electronics:
+  - Flight controller: [Holybro KakuteH7](../flight_controller/kakuteh7.md)
+  - Coupled with 2x [Tekko32 F4 4in1 ESCs](https://shop.holybro.com/tekko32-f4-4in1-50a-esc_p1283.html)
+   :::note
+   You can select your own flight controller of choice, it just needs to support 8 DShot outputs.
+   :::
+  - GPS: [ZED-F9P](https://www.gnss.store/gnss-gps-modules/105-ublox-zed-f9p-rtk-gnss-receiver-board-with-sma-base-or-rover.html?search_query=ZED-F9P&results=11)
+  - [GPS helix antenna](https://www.gnss.store/rf-gps-antennas/28-high-performance-multi-band-gnss-active-quad-helix-antenna-for-rtk.html)
+   :::note
+   Any other GPS may work as well, however a helix antenna is expected to perform better for inverted flights.
+   :::
+  - Any RC receiver
+  - An external Magnetometer. We used the [RM-3100](https://store-drotek.com/893-professional-grade-magnetometer-rm3100.html).
+  - Telemetry link, e.g. [WiFi](../telemetry/telemetry_wifi.md)
+- Propulsion:
+  - Motors: 8x [BrotherHobby LPD 2306.5 2000KV/2450KV/2650KV](https://www.getfpv.com/brotherhobby-lpd-2306-5-2000kv-2450kv-2650kv-motor.html)
+  - 3D Propellers: 2x [HQProp 3D 5X3.5X3 3-Blade Propeller (Set of 4)](https://www.getfpv.com/hqprop-3d-5x3-5x3-3-blade-propeller-set-of-4.html) or 2x [Gemfan 513D 3-Blade 3D Propeller (Set of 4)](https://www.getfpv.com/gemfan-513d-durable-3-blade-propeller-set-of-4.html)
+  - Battery: we used a 6S 3300mAh LiPo. Make sure to check the dimensions so it fits the frame.
+  - Battery strap
+- Frame:
+  - Carbon square tube R 8mm X 7mm X 1000mm, e.g. [here](https://shop.swiss-composite.ch/pi/Halbfabrikate/Rohre/Vierkant-Rohre/CFK-Vierkantrohr-8x8-7x7mm.html)
+  - Carbon Rods R 3mm X 2mm X 1000mm, e.g. [here](https://shop.swiss-composite.ch/pi/Halbfabrikate/Rohre/CFK-Rohre-pultrudiert-pullwinding/Carbon-Microtubes-100cm-x-20-3mm.html)
+- TODO: 3D parts + screws + standoffs + rod lengths
```

Comment:
rod lengths:
square tube: 8 pieces  with length of 248mm
rods: 12x328mm, 6x465mm

Reply:
Thanks, added